x/oracle/keeper/migrations.go

```diff
@@ -22,3 +23,12 @@ func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 	m.keeper.SetMaximumMedianStamps(ctx, 1)
 	return nil
 }
+
+// Migrate2to3 migrates from version 2 to 3.
+func (m Migrator) Migrate2to3(ctx sdk.Context) error {
```

Comment:
Parameter value change is not a consensus version change.  Previously we needed to upgrade the consensus version, because it was breaking.
Let's rename this function to `HistoroacleParamsV3_4` (or something like that).

Reply:
Won't this be consensus changing since we're changing how ABCI works? It definitely changes consensus for x/leverage once moser's PR gets merged

Reply:
In any case, I've changed it to just directly update those params: 
https://github.com/umee-network/umee/pull/1656/commits/1bebad098b87b03786c2f7d569c73651cd446d53